### DISCUSSION TITLE
How to connect the agent with the php application?

### DISCUSSION THREAD

**Discussion by ashu99099** [SkyAPM/SkyAPM-php-sdk#476] [Q&A] [answered]
How to connect the agent(SkyAPM-php-sdk) with the php application (that we want to monitor)?

**Comment by Isanfurg** [comment, marked as answer]
you can follow this steps
https://notes.dingzhihao.org/notes/info/12231447

**Comment by heyanlong** [comment]
https://github.com/SkyAPM/SkyAPM-php-sdk/blob/v4.1.2/docs/install.md
